tests/introspection/ApiCMAttachmentTest.cs

```diff
@@ -416,9 +425,45 @@ protected INativeObject GetINativeInstance (Type t)
 					new CVPixelBufferPoolSettings (),
 					new CVPixelBufferAttributes (CVPixelFormatType.CV24RGB, 100, 50)
 				);
+			case "NWAdvertiseDescriptor":
+				return NWAdvertiseDescriptor.CreateBonjourService ("sampleName" + DateTime.Now);
```

Comment:
The NRE in the tests is because `GetINativeInstance` returns `null` which is likely because of the extra entries being added returns `null`. It could be this one